### ISSUE TITLE
Rosewine Circle has no stats

### ISSUE THREAD

**Issue by pickel90** [elysium-project/server#926]
The topic explains it all. The item "Rosewince Circle" is blank . All it has is blue rarity with a lvl 55 requirement. Has no effect either, when equipped.

**Comment by Sabre142** [comment]
Delete your WDB folder.

**Comment by pickel90** [comment]
I've done that. The item still appears statless

**Comment by strudelkuchen** [comment]
Can confirm.

**Comment by Sabre142** [comment]
What server?

**Comment by pickel90** [comment]
Anathema

**Comment by Maxvla** [comment]
Added in later patch, I believe.

**Comment by EriksGit** [comment]
I've looked at this item using web.archive.org on wow.allakhazam.com and this is what I found.

Rosewine Circle initially had the stats +16 Spirit & +7 Agility, and dropped from War Master Voone at least as early as April 5th 2005 (a month after patch 1.3 released), as seen here: http://web.archive.org/web/20050405235019/http://wow.allakhazam.com:80/db/item.html?witem=13178
The last entry with those stats and from that boss is this one, from March 7th 2006 (during patch 1.9): http://web.archive.org/web/20060307184719/http://wow.allakhazam.com:80/db/mob.html?wmob=9237

In the next entry, from May 25th 2006 (during patch 1.10), the item was rebalanced to instead have +5 MP5 & +29 Healing, and to drop from Urok Doomhowl instead of War Master Voone: http://web.archive.org/web/20060525081919/http://wow.allakhazam.com:80/db/mob.html?wmob=10584


Here's Urok Doomhowl's loot table before patch 1.10: http://web.archive.org/web/20060307182734/http://wow.allakhazam.com:80/db/mob.html?wmob=10584
As you can see, no Rosewine Circle had been recorded to drop off of him by then.

In conclusion, before patch 1.10, Rosewine Circle should drop from War Master Voone and have the stats +16 Spirit & +7 Agility.
After patch 1.10, it should drop from Urok Doomhowl and have the stats +5 MP5 & +29 Healing.

**Comment by QAkvastskaft** [comment]
Summary from EriksGit 
Before patch 1.10, Rosewine Circle should drop from War Master Voone and have the stats +16 Spirit & +7 Agility.
After patch 1.10, it should drop from Urok Doomhowl and have the stats +5 MP5 & +29 Healing.

**Comment by QAkvastskaft** [comment]
Fix applied: https://forum.elysium-project.org/topic/56778-development-update-14102017/
